### PR TITLE
Rename "Native File System" to "File System Access"

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Rename "Native File System" to "File System Access"

### CODE REVIEW

File: src/site/content/en/blog/file-system-access/index.md [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
 
-The [Native File System API][spec]  (formerly known as the Writeable Files API)
+The [File System Access API][spec] (formerly known as Native File System API
+and prior to that Writeable Files API)
 enables developers to build powerful web apps
 that interact with files on the user's local device, like IDEs, photo and video
 editors, text editors, and more. After a user grants a web app access, this
 API allows web apps to read or save changes directly to files and folders
-on the user's device. Beyond reading and writing files, the Native File System
+on the user's device. Beyond reading and writing files, the file system access
```
Comment (tomayac) [inline src/site/content/en/blog/file-system-access/index.md:29]:
```suggestion
on the user's device. Beyond reading and writing files, the File System Access
```

File: src/site/content/en/blog/file-system-access/index.md [hunk 2]
```diff
@@ -398,15 +399,15 @@ for access to the files.
...
+We want to hear about your experiences with the File System Access API.
 
 ### Tell us about the API design {: .hide-from-toc }
 
 Is there something about the API that doesn't work like you expected? Or
 are there missing methods or properties that you need to implement your
 idea? Have a question or comment on the security model?
 
-* File a spec issue on the [WICG Native File System GitHub repo][spec-issues],
+* File a spec issue on the [WICG file system access GitHub repo][spec-issues],
```
Comment (tomayac) [inline src/site/content/en/blog/file-system-access/index.md:410]:
```suggestion
* File a spec issue on the [WICG File System Access GitHub repo][spec-issues],
```

File: src/site/content/en/blog/file-system-access/index.md [hunk 3]
```diff
@@ -432,32 +433,32 @@ critical it is to support them.
 ## Helpful links {: #helpful }
 
 * [Public explainer][explainer]
-* [Native File System specification][spec] & [File specification][file-api-spec]
+* [file system access specification][spec] & [File specification][file-api-spec]
```
Comment (tomayac) [inline src/site/content/en/blog/file-system-access/index.md:436]:
```suggestion
* [File System Access specification][spec] & [File specification][file-api-spec]
```

File: src/site/content/en/progressive-web-apps/app-like-pwas/index.md [hunk 4]
```diff
@@ -228,9 +228,9 @@ Other native applications and even websites or emails can integrate with the Pod
   for PWAs. Currently, <a href="https://developer.mozilla.org/en-US/docs/Web/API/Navigator/registerProtocolHandler"><code>registerProtocolHandler()</code></a> with a <code>web+</code> scheme prefix is the best alternative.
 {% endDetails %}
 
-## Native file system integration
+## File system access integration
```
Comment (tomayac) [inline src/site/content/en/progressive-web-apps/app-like-pwas/index.md:231]:
```suggestion
## File system integration
```

File: src/site/content/en/progressive-web-apps/app-like-pwas/index.md [hunk 5]
```diff
@@ -584,10 +584,10 @@ The table below shows a compact overview of all features and provides a list of
         </td>
       </tr>
       <tr>
-        <td><a href="#native-file-system-integration">Native file system integration</a></td>
+        <td><a href="#file-system-access-integration">file system access integration</a></td>
```
Comment (tomayac) [inline src/site/content/en/progressive-web-apps/app-like-pwas/index.md:587]:
```suggestion
        <td><a href="#file-system-access-integration">Local file system integration</a></td>
```

File: src/site/content/en/blog/browser-nativefs/index.md [hunk 6]
```diff
@@ -164,19 +166,19 @@ const saveFile = async (blob) => {
 
 ## Introducing browser-nativefs
 
-As perfectly fine as the Native File System API is,
+As perfectly fine as the File System Access API is,
 it's [not yet  widely available](https://caniuse.com/#feat=native-filesystem-api).
```
Comment (beaufortfrancois) [inline src/site/content/en/blog/browser-nativefs/index.md:170]: You may want to update caniuse data as well.

File: src/site/content/en/blog/browser-nativefs/index.md [hunk 7]
```diff
@@ -164,19 +166,19 @@ const saveFile = async (blob) => {
...
 
 <figure class="w-figure">
   <img class="w-screenshot" src="caniuse.png"
-       alt="Browser support table for the Native File System API. All browsers are marked as 'no support' or 'behind a flag'.">
+       alt="Browser support table for the File System Access API. All browsers are marked as 'no support' or 'behind a flag'.">
   <figcaption class="w-figcaption">
-    Browser support table for the Native File System API.
+    Browser support table for the File System Access API.
     (<a href="https://caniuse.com/#feat=native-filesystem-api">Source</a>)
```
Comment (beaufortfrancois) [inline src/site/content/en/blog/browser-nativefs/index.md:177]: Same here for caniuse

File: src/site/content/en/blog/file-system-access/index.md [hunk 8]
```diff
@@ -1,40 +1,41 @@
...
+description: The new File System Access API enables developers to build powerful web apps that interact with files on the user's local device, like IDEs, photo and video editors, text editors, and more. After a user grants a web app access, this API allows web apps to read or save changes directly to files and folders on the user's device.
 date: 2019-08-20
 updated: 2020-09-02
 tags:
   - blog
   - capabilities
   - file
   - file-system
-  - native-file-system
+  - file-system-access
```
Comment (beaufortfrancois) [inline src/site/content/en/blog/file-system-access/index.md:14]: Why is this a tag in the first place?

File: src/site/content/en/blog/fugu-status/index.md [hunk 9]
```diff
@@ -254,12 +254,13 @@ latest version of Chrome, and in many cases other Chromium based browsers.
...
+            File System Access API
           </a>
         </td>
         <td>
-          The Native File System API (formerly known as the Writable Files API)
+          The File System Access API (formerly known as Native File System API
+          and prior to that Writable Files API)
           enables developers to build powerful web apps that interact with files
           on the user's local device, like IDEs, photo and video editors, text
           editors, and more. After a user grants a web app access, this API
```
Comment (beaufortfrancois) [inline src/site/content/en/blog/fugu-status/index.md:266]: You may want to update fugu-status page updated and the updated date of this feature as well?

File: src/site/content/en/blog/async-clipboard/index.md [hunk 10]
```diff
@@ -228,7 +228,7 @@ match style** or **Paste without formatting**.
 The following example shows how to do this. This example uses `fetch()` to obtain
 image data, but it could also come from a
 [`<canvas>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/canvas)
-or the [Native File System API](https://web.dev/native-file-system/).
+or the [File System Access API](https://web.dev/file-system-access/).
```
Comment (tomayac) [inline src/site/content/en/blog/async-clipboard/index.md:231]:
```suggestion
or the [File System Access API](/file-system-access/).
```

[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/async-clipboard/index.md:231]: This was there, but I remove it now.

File: src/site/content/en/blog/browser-nativefs/index.md [hunk 11]
```diff
@@ -101,20 +103,20 @@ open→edit→save flow happen, that is, there is no way to *overwrite* the orig
...
-The Native File System API makes both operations, opening and saving, a lot simpler.
+The File System Access API makes both operations, opening and saving, a lot simpler.
 It also enables *true saving*, that is, you can not only choose where to save a file,
 but also overwrite an existing file.
 
 {% Aside %}
-  For a more thorough introduction to the Native File System API, see the article
-  [The Native File System API: simplifying access to local files](https://web.dev/native-file-system/).
+  For a more thorough introduction to the File System Access API, see the article
+  [The File System Access API: simplifying access to local files](https://web.dev/file-system-access/).
```
Comment (tomayac) [inline src/site/content/en/blog/browser-nativefs/index.md:114]:
```suggestion
  [The File System Access API: simplifying access to local files](/file-system-access/).
```

File: src/site/content/en/blog/browser-nativefs/index.md (hunk 6 above)
Comment (tomayac) [reply src/site/content/en/blog/browser-nativefs/index.md:170]: Yes, I thought about this. Will do.

File: src/site/content/en/blog/file-system-access/index.md (hunk 8 above)
Comment (tomayac) [reply src/site/content/en/blog/file-system-access/index.md:14]: No idea, Pete put it there.

File: src/site/content/en/blog/browser-nativefs/index.md (hunk 7 above)
Comment (tomayac) [reply src/site/content/en/blog/browser-nativefs/index.md:177]: Yepp

File: src/site/content/en/blog/file-system-access/index.md (hunk 8 above)
Comment (tomayac) [inline src/site/content/en/blog/file-system-access/index.md:14]:
```suggestion
```

Comment (tomayac) [reply src/site/content/en/blog/file-system-access/index.md:14]: Lets kill it

File: src/site/content/en/blog/fugu-status/index.md (hunk 9 above)
Comment (tomayac) [reply src/site/content/en/blog/fugu-status/index.md:266]: Will do

File: src/site/content/en/blog/browser-nativefs/index.md (hunk 6 above)
Comment (tomayac) [inline src/site/content/en/blog/browser-nativefs/index.md:170]:
```suggestion
it's [not yet  widely available](https://caniuse.com/#feat=file-system-access-api).
```

Context: https://github.com/Fyrd/caniuse/pull/5622

File: src/site/content/en/blog/browser-nativefs/index.md (hunk 7 above)
Comment (tomayac) [inline src/site/content/en/blog/browser-nativefs/index.md:177]:
```suggestion
    (<a href="https://caniuse.com/#feat=file-system-access-api">Source</a>)
```

Context: https://github.com/Fyrd/caniuse/pull/5622.

**Comment by tomayac** [comment]
Also reflected in https://github.com/Fyrd/caniuse/pull/5622.

**Comment by tomayac** [comment]
Also reflected in https://github.com/mozilla/standards-positions/pull/442.

**Review by robdodson — APPROVED** [review]
Changes to .js and .yaml LGTM

**Review by kaycebasques — APPROVED** [review]
Woohoo many thanks for doing this

**Review by kaycebasques — CHANGES_REQUESTED** [review]
@robdodson please check out that visual regression on the authors page… looks fishy.

**Comment by robdodson** [comment]
that's really odd. I'm going to push an empty commit to see if percy will run again.

**Comment by robdodson** [comment]
@kaycebasques seems to be fixed. The broken image is a recurring issue that we need to work with the Percy folks to debug.

**Review by kaycebasques — APPROVED** [review]

**Pull request merged**